plugins/auto-refetch/src/index.ts

```diff
@@ -0,0 +1,107 @@
+import type { PiniaColadaPlugin, UseQueryOptions } from '@pinia/colada'
+import type { MaybeRefOrGetter } from 'vue'
+import { toValue } from 'vue'
+
+export interface PiniaColadaAutoRefetchOptions {
+  /**
+   * Whether to enable auto refresh by default.
+   * @default false
+   */
+  enabled?: boolean
```

Comment:
let's rename it to match the per-query option
```suggestion
  autoRefetch?: boolean
```